ingestors/java/spring-reactive/spring-reactive-client/src/main/kotlin/com/metlo/configuration/MetloConfigurationProperties.kt

```diff
@@ -0,0 +1,45 @@
+package com.metlo.configuration
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+@ConstructorBinding
+@ConfigurationProperties("metlo", ignoreUnknownFields = true)
+data class MetloConfigurationProperties(
+    /**
+     * Enable Metlo request and response tracing.
+     */
+    val enabled: Boolean = true,
+    /**
+     * The maximum number of threads to use for asynchronous communication.
+     */
+    val threadPoolSize: Int = 2,
+    /**
+     * The maximum number of trace requests per second to capture, -1 means unlimited.
+     */
+    val requestsPerSecond: Int = -1,
+    /**
+     * The Metlo connection API key to use for authorization.
+     */
+    val apiKey: String = "",
+    /**
+     * The Metlo API host URL to send traces to.
+     */
+    val host: String = "https://app.metlo.com:8081",
+    /**
+     * Case-insensitive collection of field names to omit from headers, bodies, and query arguments.
+     */
+    val protectedKeys: List<String> = listOf("password", "apiKey", "secret"),
```

Comment:
Can we leave this as an empty list to start with ?

Reply:
Yes, I forgot to clean that out. Will push a fix.

Reply:
Fixed